examples/dht_node.rs

```diff
@@ -152,6 +152,15 @@ fn main() {
         })
         .map_err(|_err| Error::new(ErrorKind::Other, "NatPing timer error"));
 
+    // 3 seconds for NatPingRequest
```

Comment:
3 for nat?

Reply:
https://zetok.github.io/tox-spec/#hole-punching

NAT ping requests are sent every 3 seconds in toxcore. If no response is received for 6 seconds, the hole-punching will stop.

Reply:
That's right, but the next line is `let remove_wakeup`.